jbpm-wb-common/jbpm-wb-common-client/src/main/java/org/jbpm/workbench/common/client/list/AbstractMultiGridPresenter.java

```diff
@@ -32,6 +32,9 @@
 
 public abstract class AbstractMultiGridPresenter<T extends GenericSummary, V extends MultiGridView> extends AbstractScreenListPresenter<T> implements RefreshSelectorMenuBuilder.SupportsRefreshInterval {
 
+    public static final String SEARCH_PARAMETER_PROCESS_INSTANCE_ID = "processInstanceId";
+    public static final String SEARCH_PARAMETER_PROCESS_DEFINITION_ID = "processDefinitionId";
```

Comment:
@rbarriuso I agree we need a better place to keep this, although `PerspectiveIds` doesn't seem the right place to me as we share these parameters only across the runtime views. Another point is that I believe the same parameter could be reused across different screens, so appending the Perspective Id wouldn't be ideal. I end up extending the `PerspectiveIds`, would that be a better approach?